openstack/ae-environment.yml

```diff
@@ -0,0 +1,89 @@
+heat_template_version: 2013-05-23
+description: Aeternity environment
+
+parameters:
+    environment:
+        type: string
+        label: Nodes Environment
+    node_count:
+        type: number
+        label: Nodes Count
+        description: Number of epoch nodes
+        default: 2
+        constraints:
+          - range: { min: 2 }
+            description: Nodes Count must be at least 2
+    node_flavor:
+        type: string
+        label: Node Instance Type
+        description: Type of instance (flavor) to deploy for the epoch node
+        default: c1.small
```

Comment:
Please set instances with at least 2 CPUs in order to catch some races.

Reply:
And please add comment alongside this line with number of CPUs and amount of memory.

Reply:
The default has been changed to [c1.medium](https://github.com/aeternity/infrastructure/pull/5/files#diff-011895ba49cd197cd207e4c076fc14c1R20) 4GB/2VCPU

Reply:
I added [flavor list in the playbook](https://github.com/aeternity/infrastructure/pull/5/files#diff-79a41970fbd43cf7b5f987e3410933acR2) as i find it more appropriate place.

However, I'm still not sure it's good to list a cloud provider specific parameters here.